intro/index.html

```diff
@@ -0,0 +1,163 @@
+        <script>
+            var link = document.createElement( 'link' );
+            link.rel = 'stylesheet';
+            link.type = 'text/css';
+            link.href = window.location.search.match( /print-pdf/gi ) ? '../../css/print/pdf.css' : '../../css/print/paper.css';
+            document.getElementsByTagName( 'head' )[0].appendChild( link );
+        </script>
+    </head>
+    <body>
+        <div class="reveal">
+            <div class="slides">
+                <section>
+                    <section data-background="#111111">
+
+                        <h1>Podstawy C++</h1>
+                        <a href="https://coders.school">
+                            <img width="500" data-src="../coders_school_logo.png" alt="Coders School" class="plain">
+                        </a>
+                        <h3>Mateusz Adamski</h3>
+                        <h3>Łukasz Ziobroń</h3>
+
+                    </section>
+                    <section data-markdown>
+                        <textarea data-template>
+                        <!-- .slide: style="font-size: 0.9em" -->
+
+                        ## Agenda
+
+                        * <!-- .element: class="fragment fade-in" --> Typy danych
+                        * <!-- .element: class="fragment fade-in" --> Funkcje
+                        * <!-- .element: class="fragment fade-in" --> Instrukcje warunkowe
+                        * <!-- .element: class="fragment fade-in" --> Pętle
+                        * <!-- .element: class="fragment fade-in" --> Tablice
+                        * <!-- .element: class="fragment fade-in" --> STL - co to?
+                        * <!-- .element: class="fragment fade-in" --> <code>std::vector</code>
+                        * <!-- .element: class="fragment fade-in" --> Pętla <code>for</code> po kolekcji
+                        * <!-- .element: class="fragment fade-in" --> <code>std::string</code>
+                        * <!-- .element: class="fragment fade-in" --> <code>std::list</code>
+                        * <!-- .element: class="fragment fade-in" --> <code>std::map</code>
+                        * <!-- .element: class="fragment fade-in" --> Zakresy (scopes)
+                        * <!-- .element: class="fragment fade-in" --> Referencje
+                        * <!-- .element: class="fragment fade-in" --> Wskaźniki
+                        * <!-- .element: class="fragment fade-in" --> Zagrożenia przy stosowaniu referencji i wskaźników
+                        * <!-- .element: class="fragment fade-in" --> Typ wyliczeniowy <code>enum</code> i <code>enum class</code>
+
+                        </textarea>
+                    </section>
+                    <section data-markdown>
+                        <textarea data-template>
+
+                        ## Zadania
+
+                        [Repo GH `coders-school/cpp-fundamentals`](https://github.com/coders-school/cpp-fundamentals/tree/master/module1/)
```

Comment:
```suggestion
                        [Repo GH `coders-school/cpp-fundamentals`](https://github.com/coders-school/cpp-fundamentals/)
```
Podlinkujmy tutaj całe repo, nie tylko module1